src/Javis.jl

```diff
@@ -941,6 +941,12 @@ function javis(
         ffmpeg_exe(`-loglevel panic -framerate $framerate -i $(tempdirectory)/%10d.png -i
                     "$(tempdirectory)/palette.bmp" -lavfi
                     "paletteuse=dither=sierra2_4a" -y $pathname`)
+    elseif ext == ".mp4"
+        # generate a colorpalette first so ffmpeg does not have to guess it
+        ffmpeg_exe(`-loglevel panic -i $(tempdirectory)/%10d.png -vf
+                    "palettegen=stats_mode=diff" -y "$(tempdirectory)/palette.bmp"`)
+	ffmpeg_exe(`-loglevel panic -i $(tempdirectory)/%10d.png -framerate $framerate -c:v libx264 -r 30 -pix_fmt yuv420p $pathname`)
```

Comment:
this line has a different indentation. Seems like that this occurs often. My vs code uses 4 spaces.

More importantly this does not seem to work for me. I had to change it to:
```
-loglevel panic -framerate $framerate -i $(tempdirectory)/%10d.png -c:v libx264 -pix_fmt yuv420p $pathname
```
so `-framerate` needs to be before `-i` and I dropped `-r` as I think this might be confusing.